SETUP.md

```diff
@@ -188,11 +188,87 @@ packages/files, and thus probably don't use the debian image.
 
 ## Clean-up
 
-If you'd like to delete the nodes in your database, you execute the query:
+To delete the nodes in your database, you execute the query:
 ```
 MATCH (n) DETACH DELETE n;
 ```
 
+# Example 3: OSV Certifier
```

Comment:
q: do we need to update the indexing queries to index to include these nodes as well or has that been done.

Reply:
Index for the vulnerability nodes have been added: https://github.com/guacsec/guac/blob/main/cmd/guacone/cmd/files.go#L255-L262